00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+Enarx project organizes its work.
+
+Currently, the project uses multiple area-specific boards. This proposal
+abandons that idea and instead uses a single, unified project board that
+contains the entire project backlog and all current work.
+
+The board will be organized with the following columns:
+
+- backlog: collection of issues not currently being worked on
+- nominated: issues nominated by team members for the next sprint (planning)
+- accepted: issues the team should work on next sprint (planning)
+- assigned: issues actively being worked on by an assigned team member
+- under review: all submitted PRs go here while awaiting review
+- done: all issues and PRs go here when done or merged, respectively
+
+Here's a quick diagram:
+
+![Complete issue workflow diagram](./project-board.svg)
+
+In more technical terms, this would be an organization-level Github project
+board. This will allow for issues and PRs across _any_ Enarx repository (not
+just the main one) to be included, which supports modularization of the project
+as it grows.
+
+## Meetings
+
+The concepts and processes introduced above are put into action in the form of
+several team meetings that happen throughout the process of a single sprint.
+
+To start, consider this high-level overview of how an issue progresses through
+the process, beginning with its opening and continuing all the way until it's
+considered done:
+
+![Complete issue workflow diagram](./full-process.svg)
+
+This flowchart introduces the three crucial events of the process, each of
+which is encapsulated by its own team meeting:
+ 
+ - **Planning**, a meeting representing the start of a new sprint
+ - **Daily Standup**, the team's daily meeting where, among other things,
+   **Triage** is performed.
+ - **Retrospective**, or "Retro" for short, a meeting representing the
+   end of a sprint
+
+Once an issue is filed, it will be categorized and discussed in Triage. If
+urgent, it's injected straight into the current sprint; if not, it gets pushed
+to the backlog. From there, the rest of the cycle will carry it through to
+completion: Sprint Planning will sooner or later land it as part of a sprint,
+where it will become a team member's action item to complete. At the end of the
+sprint, Sprint Retro will allow the team to acknowledge its completion or need
+for extra time as appropriate. When finished, the issue is considered complete.
+
+The following sections will dive deep into the process of each meeting,
+carefully considering actions that need to be taken, who should be responsible
+for those actions, and how automation might be able to aid those actions.
+
+### Planning
+
+Sprint planning constitutes the beginning of a new sprint. It's where the team
+and project leadership collectively decide what the project's focus should be.
+
+The sprint planning process is illustrated in the following flowchart:
+
+![Sprint Planning Diagram](./sprint-planning.svg)
+
```

Comment:
This is a detail, but what do the diamonds vs. squares mean?

Reply:
They're standard (ish) ISO flowchart shapes: https://en.wikipedia.org/wiki/Flowchart

I've added an appendix that explains what they mean in the context of this RFC. Pushing soon.